### PR TITLE
Use channels to pass data to the debug tui  

### PR DESCRIPTION
**Pull request by eharris128**
What
===============
- Initial [commit](https://github.com/mintoolkit/mint/commit/67b46987b35539188fad62ffa23a297d825e8bd9) removed the the data  channel from the image command's handler.go and added a Data method to execution context.
  - This approach can likely be deprecated now in favor of the approach taken for passing data to the `debug` tui via an internal channel on execution context
- Move the debug tui to be a sibling file to handler.go
- Add tui hotkeys to support interaction with the debug tui
- Add an internal channel on execution context to feed data from `.Info` to a provided `dataChannel` via an `internalDataCh` [note - channel naming should be consistent]
  - When the `OutputFormat` for execution context is specified as `"subscription"` - then the channels are used to pipe data. This is only implemented for `.Info` at the moment for this Proof of Concept.

Why
===============
- Demonstrate PoC of using an internal channel inside of execution context
- Begin to support a tui mode for debug

## Questions
- The approach to support `debug --tui` effectively drops all other provided flags on the floor. This could be a poor UX.
  - Will the pattern be to encourage people to only use the `--tui` flag as a standalone flag, then perform interactions inside of the interface?
  - Or will `debug --tui --runtime=k8s --target=my-nginx` be a desired entrypoint to support? DevX wise - for ease of using the last invoked command without writing out things by hand in a TUI, supporting flags alongside --tui feels relevant.

How Tested
===============
`mint debug --tui`
`mint images --tui`
`mint tui` -> `d`

### CODE REVIEW

File: pkg/app/execontext.go [hunk 1]
```diff
@@ -272,6 +287,15 @@ func (ref *Output) Message(data string) {
 
 }
 
+func (ref *Output) Data(channelKey string, data interface{}) {
+	if ch, exists := ref.DataChannels[channelKey]; exists {
+		ch <- data // Send data to the corresponding channel
+		fmt.Printf("Data sent to channel '%s': %v\n", channelKey, data)
```
Comment (kcq) [inline pkg/app/execontext.go:293]: let's use logrus log instead of fmt.Printf

File: pkg/app/execontext.go [hunk 2]
```diff
@@ -272,6 +287,15 @@ func (ref *Output) Message(data string) {
...
+	} else {
+		fmt.Printf("Channel for channelKey '%s' not found\n", channelKey)
```
Comment (kcq) [inline pkg/app/execontext.go:295]: let's use logrus log instead of fmt.Printf

File: pkg/app/execontext.go (hunk 1 above)
Comment (eharris128) [reply pkg/app/execontext.go:293]: Noted. Will update / remove.

File: pkg/app/execontext.go (hunk 2 above)
Comment (eharris128) [reply pkg/app/execontext.go:295]: Noted. Will update / remove.

**Pull request merged**